app/containers/List/ListItem.js

```diff
@@ -46,47 +43,68 @@ const styles = StyleSheet.create({
 		...sharedStyles.textRegular
 	},
 	actionIndicator: {
-		...I18nManager.isRTL
-			? { transform: [{ rotate: '180deg' }] }
-			: {}
+		...(I18nManager.isRTL ? { transform: [{ rotate: '180deg' }] } : {})
 	}
 });
 
-const Content = React.memo(({
-	title, subtitle, disabled, testID, left, right, color, theme, translateTitle, translateSubtitle, showActionIndicator, fontScale
-}) => (
-	<View style={[styles.container, disabled && styles.disabled, { height: BASE_HEIGHT * fontScale }]} testID={testID}>
-		{left
-			? (
-				<View style={styles.leftContainer}>
-					{left()}
-				</View>
-			)
-			: null}
-		<View style={styles.textContainer}>
-			<Text style={[styles.title, { color: color || themes[theme].titleText }]} numberOfLines={1}>{translateTitle ? I18n.t(title) : title}</Text>
-			{subtitle
-				? <Text style={[styles.subtitle, { color: themes[theme].auxiliaryText }]} numberOfLines={1}>{translateSubtitle ? I18n.t(subtitle) : subtitle}</Text>
-				: null
-			}
-		</View>
-		{right || showActionIndicator
-			? (
+const Content = React.memo(
+	({
+		title,
+		subtitle,
+		disabled,
+		testID,
+		left,
+		right,
+		color,
+		theme,
+		translateTitle,
+		translateSubtitle,
+		showActionIndicator,
+		fontScale
+	}) => (
+		<View
+			style={[
+				styles.container,
+				disabled && styles.disabled,
+				{ height: BASE_HEIGHT * fontScale }
+			]}
+			testID={testID}
+		>
+			{left ? <View style={styles.leftContainer}>{left()}</View> : null}
+			<View style={styles.textContainer}>
+				<Text
+					style={[styles.title, { color: color || themes[theme].titleText }]}
+					numberOfLines={1}
+				>
+					{translateTitle ? I18n.t(title) : title}
+				</Text>
+				{subtitle ? (
+					<Text
+						style={[styles.subtitle, { color: themes[theme].auxiliaryText }]}
+						numberOfLines={1}
+					>
+						{translateSubtitle ? I18n.t(subtitle) : subtitle}
+					</Text>
+				) : null}
+			</View>
+			{right || showActionIndicator ? (
 				<View style={styles.rightContainer}>
 					{right ? right() : null}
-					{showActionIndicator ? <Icon name='chevron-right' style={styles.actionIndicator} /> : null}
+					{showActionIndicator ? (
+						<Icon name='chevron-right' style={styles.actionIndicator} />
+					) : null}
 				</View>
-			)
-			: null}
-	</View>
-));
+			) : null}
+		</View>
+	)
+);
 
-const Button = React.memo(({
-	onPress, ...props
-}) => (
+const Button = React.memo(({ onPress, backgroundColor, ...props }) => (
```

Comment:
Since you're adding a background color, you should also add the option to overwrite `underlayColor` from `Touch`.
Sending it as a prop should work as `Touch` has `{...props}`, but be aware not to change the behaviour when the prop isn't sent.

After this, add the new colors to `colors.js` (We're going to refactor it as soon as we get themes on web client).
I suggest the name `tintActive`.
Light: `#549DF9`
Dark: `#549DF9`
Black: `#76B7FC`

It should behave like this:
![Jan-12-2021 18-53-12](https://user-images.githubusercontent.com/804994/104378921-77ebb480-5507-11eb-8821-49b6e0d55b5d.gif)

Reply:
Don't forget to add stories for the new props.